compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -317,6 +327,24 @@ execCompile inputFile outputFile opts = withProjectRoot' (ProjectOpts Nothing (P
         createDirectoryIfMissing True $ takeDirectory outputFile
         B.writeFile outputFile $ Archive.encodeArchive bs
 
+execLint :: FilePath -> Options -> Command
+execLint inputFile opts =
+  withProjectRoot' (ProjectOpts Nothing (ProjectCheck "" False)) $ \relativize ->
+  do
+    loggerH <- getLogger opts "lint"
+    inputFile <- toNormalizedFilePath <$> relativize inputFile
+    opts' <- mkOptions opts
+    defaultHlintDataDir <-locateRunfiles $ mainWorkspace </> "compiler/damlc/daml-ide-core"
+    -- If not `--with-hlint path` given on the command line, assume
+    -- the testing default.
+    let opts'' =
+          case optHlintUsage opts' of
```

Comment:
I wonder if we’re handling this the right way. We probably want somewhat similar logic for all commands (in particular for the IDE) but then users cannot actually disable HLint fully. Of course for an explicit `lint` command that doesn’t make sense anyway but for the IDE it does. Maybe we want to differentiate between explicitly disabled and not specified? Just a thought, no need to do anything about this as part of this PR.

Reply:
Ok.